AfricanReligions/decisions/afr_decisions.txt

```diff
@@ -750,4 +750,64 @@ title_decisions = {
 			factor = 0
 		}
 	}
-}
\ No newline at end of file
+	
+	pick_religious_tradition = {
+		only_playable = yes
+		is_high_prio = yes
+		ai_check_interval = 12
+		potential = {
+			has_dlc = "The Old Gods"
+			is_playable = yes
+			is_adult = yes
+			prisoner = no
```

Comment:
I don't think this should be in the potential block.  We want the decision to still be visible even if you're imprisoned, just have it display that you can't take it right now.